source/ngram-full-text-search.rst

```diff
@@ -0,0 +1,37 @@
+.. full-text-search.ngram:
+
+================================================================================
+|ngram| Full Text Search
+================================================================================
+
+The |ngram| full text search is useful for searching text for a
+specific string of characters in a field of a collection. To use |ngram|, create
+a text index on a collection setting the ``default_language``
+parameter to **ngram**:
+
+.. code-block:: guess
+
+   mongo > db.collection.createIndex({name:"text"}, {default_language: "ngram"})
+
+The |ngram| search engine uses the following characters that not used as parts
```

Comment:
This is strange and not sure what it means. Probably better would be "The ngram search engine uses the following characters as tokenizer delimiters:"

Reply:
Thanks for noting this problem. 'tokenizer delimiter' (tokenizer usually represents a tool which extracts tokens) sounds suspicious and i didn't find it used in reliable contexts. Using 'token delimiter character'.